lib/button-control/inc/button.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <Arduino.h>
+const unsigned long debounce_delay = 75;
+const int high_button_state = HIGH;
+const unsigned long medium_press = 1000;
+const unsigned long long_press = 5000;
+
+enum Press {
+    NOT_PRESSED = 0,
+    PRESSED_SHORT = 1,
+    PRESSED_MEDIUM = 2,
+    PRESSED_LONG = 3
+};
+
+class Button
+{
+private:
+    short pinNum;
```

Comment:
- There are two types of the cases. `pinNum` and `press_time` . Chose one and implement to others :D `camelCase` or `snake_case`

- Use explicite types of `ints`. https://en.cppreference.com/w/cpp/types/integer . The `pinNum` can be `uint8_t` - unsigned int 8 bit. 0 - 255.